articles/azure-functions/durable-functions-checkpointing-and-replay.md

```diff
@@ -86,15 +86,15 @@ A few notes on the column values:
     * **TaskScheduled**: An activity function was scheduled. The name of the activity function is captured in the `Name` column.
     * **TaskCompleted**: An activity function completed. The result of the function is in the `Result` column.
     * **TimerCreated**: A durable timer was created. The `FireAt` column contains the scheduled UTC time at which the timer expires.
-    * **TimerFired**: A durable timer expired.
+    * **TimerFired**: A durable timer fired.
     * **EventRaised**: An external event was sent to the orchestration instance. The `Name` column captures the name of the event and the `Input` column captures the payload of the event.
     * **OrchestratorCompleted**: The orchestrator function awaited.
     * **ContinueAsNew**: The orchestrator function completed and restarted itself with new state. The `Result` column contains the value, which is used as the input in the restarted instance.
     * **ExecutionCompleted**: The orchestrator function ran to completion (or failed). The outputs of the function or the error details are stored in the `Result` column.
 * **Timestamp**: The UTC timestamp of the history event.
 * **Name**: The name of the function that was invoked.
 * **Input**: The JSON-formatted input of the function.
-* **Output**: The output of the function; that is, its return value.
+* **Result**: The JSON-formatted output of the function; that is, its return value.
```

Comment:
Specify that output is serialized to JSON:
"Return values are serialized to JSON" (https://docs.microsoft.com/en-us/azure/azure-functions/durable-functions-bindings#trigger-behavior)

Reply:
Changing from **Output** to **Result** is correct, but it's not necessarily the case that the value of this column will be JSON-formatted. For example, in the case of termination or failure, the column value will be a simple string. I would just leave out the "JSON-formatted" note for the same of this doc update.

Reply:
Change has been completed.